tests/index.js

```diff
@@ -732,6 +732,36 @@ describe('Thorn', () => {
 
                 expect(server.isDone()).to.be.true;
             });
+
+            it('should use the internal username for server interactions', function*() {
```

Comment:
I've created https://github.com/sugarcrm/thorn/issues/142 to address this issue.